src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/softmax/softmax_items_class_kernel_base.cpp

```diff
@@ -50,6 +51,8 @@ std::vector<size_t> SoftmaxItemsClassKernelBase::GetSoftmaxDimGlobalSizes(Softma
             return {out.X().v * out.Y().v, out.Feature().v, out.Batch().v};
         case SoftmaxDim::FEATURE:
             return {out.X().v * out.Z().v, out.Y().v, out.Batch().v};
+        case SoftmaxDim::BATCH:
+            return {out.X().v * out.Z().v, out.Y().v, out.Feature().v};
```

Comment:
can I ask why this is not iterating for Z after X ? 
I think if the layout if bfyx format, iterating x => y => z => f will be better for cache locality?

Reply:
Could you please explain how this case is different from the previous one (lines 52-53) in this regard?

From the first glance the same arguing applies for SoftmaxDim::FEATURE as well, so maybe they both shall be fixed together. Please correct me if I'm wrong.

Reply:
Closing this comment since it is not that important matter at this moment and seems that it requires other cases to be fixed too.